waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -557,7 +557,7 @@ private static long getHighWaterMark(int partitionId, WaltzClientConfig config)
      * A transaction callback to help construct {@link WaltzClient}. It is dummy because
      * it is not suppose to receive any callbacks.
      */
-    private static final class DummyTxnCallbacks implements WaltzClientCallbacks {
+    public static final class DummyTxnCallbacks implements WaltzClientCallbacks {
```

Comment:
I agree that we are creating a Private DummyTxnCallbacks class in each CLI separately which is redundancy. 
But instead of using the `DummyTxnCallbacks` class from ClientCli in PerformanceCli doesn't seem the right way to do it. 

I was thinking may be have a separate utilityCli or UtilsCli that can contains the common implementations across the CLI's and move the code over there.